h2o-algos/src/main/java/hex/tree/gbm/GBMModel.java

```diff
@@ -92,4 +96,22 @@ public GBMParameters() {
       body.ip("hex.genmodel.GenModel.correctProbabilities(preds, PRIOR_CLASS_DISTRIB, MODEL_CLASS_DISTRIB);").nl();
     body.ip("preds[0] = hex.genmodel.GenModel.getPrediction(preds, PRIOR_CLASS_DISTRIB, data, " + defaultThreshold() + ");").nl();
   }
+
+
+  @Override
+  public Model<GBMModel, GBMModel.GBMParameters, GBMModel.GBMOutput>.RawDataStreamWriter getRawDataStream() {
+    return new GBMModel.RawDataStreamWriter();
+  }
+
+  public class RawDataStreamWriter
+          extends SharedTreeModel<GBMModel, GBMModel.GBMParameters, GBMModel.GBMOutput>.RawDataStreamWriter {
+    @Override
+    protected void writeExtraModelInfo() throws IOException {
+      super.writeExtraModelInfo();
+      writeln("distribution = " + _parms._distribution);
```

Comment:
It would be nice to have this code generic - use GBMParam object to generate this info automatically via , for example, reflection.